Documentation/ceph-csi-drivers.md

```diff
@@ -138,3 +138,24 @@ csi_liveness 1
 
 Check the [monitoring doc](ceph-monitoring.md) to see how to integrate CSI
 liveness and grpc metrics into ceph monitoring.
+
+### Dynamically Expand Volume
+
+#### Prerequisite
```

Comment:
Is this whole section prerequisites? The last paragraph is how they change it instead of being a prerequisite, right?

Reply:
have only one pre-req is kubernetes version, other two points I reworded